### PR TITLE
README.md: Add an abstract

### PR DESCRIPTION
**Pull request by figalit**
I added an abstract, feel free to modify it.

### CODE REVIEW

**Review by sjackman — CHANGES_REQUESTED** [review]
Thanks, Figali! Rather than a separate file, could you please add the abstract directly to the README.md?

File: README.md [hunk 1]
```diff
@@ -4,7 +4,17 @@ Genome sequencing yields many short reads of DNA from a genome. Genome assembly
 
 We will work together to create modular assembler components for AssemblerFlow, a tool which builds pipelines of tools for genome assembly using Nextflow. Each participant will create and run a genome assembly pipeline using AssemblerFlow. We will assess the quality of each assembly, using Quast. Finally, we'll create a leader board of awesomeness to compare the assembly results!
 
-You can find our abstract [here](https://github.com/hackseq/modular-assembly-hs18/blob/master/abstract.md)
+# Abstract
+
+### Background
+Genome sequencing yields many short reads of DNA from a genome. Genome assembly attempts to reconstruct the original genome from these reads. Most genome assembler software tools are pipelines of many stages. These tools all work towards the same goal using different methods, and new tools allow for interchangeability between all the different stages of different assemblers.
+
```
Comment (sjackman) [inline README.md:11]: I suggest changing interchangeability to interoperability.

**Comment by figalit** [comment]
Sure, how about now?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,17 @@ Genome sequencing yields many short reads of DNA from a genome. Genome assembly
...
+### Problem
+It is not common to mix and match stages from all the different assembly tools into one and understand the benefits or the disadvantages of doing that, specifically towards understanding the improvement of the assembly quality. Therefore our Hackseq project aims to create a better assembly tool by mixing and matching components of various assembly tools, and visualising the results of this work. The assembly tools that we are considering are displayed [here](https://github.com/hackseq/modular-assembly-hs18/issues/2).
+
+### Solution
+We have  attempted to solve this problem by creating modular assembler components for AssemblerFlow, a tool which builds pipelines of tools for genome assembly using Nextflow, and we have created components for a number of assembly tools.
+
```
Comment (sjackman) [inline README.md:17]: Please delete this extra blank line.

Comment (figalit) [reply README.md:17]: Done.

**Review by sjackman — APPROVED** [review]

**Pull request merged**